scss/mixins/_pagination.scss

```diff
@@ -11,17 +11,63 @@
 
 // scss-docs-start pagination-max-items-mixin
 // Boosted mod
-// Ensure we only display `n` items:
-// - the first `n/2` items
-// - the last `n/2` items
-// - including the .active one (so maybe not the `n/2`th last item)
+// The pagination list has (n) items and is built in 3 parts:
+// - the begining of the list is made of the first (n/2) items
+// - the end of the list is made of the first (n/2) items
+// - the last part of the list is made of the active element (so maybe not the (n/2)th last item)
 @mixin pagination-max-items($pagination-max-items) {
-  display: none;
+  $half-items: $pagination-max-items * .5;
 
-  &.active,
-  &:nth-child(-n+#{$pagination-max-items * .5}),
-  &:nth-last-child(-n+#{$pagination-max-items * .5}) {
-    display: list-item;
+  // rule 0:
+  //  - adding the (n/2) first items
+  //  - adding the (n/2 - 1) last items
+  //  - adding the (n/2) last item if the list contains exactly (n) items
+  &:nth-child(-n+#{$half-items}),
+  &:nth-last-child(-n+#{$half-items - 1}),
+  &:nth-child(#{$half-items + 1}):nth-last-child(#{$half-items}) {
+    display: flex;
+  }
+
+  // Pagination with more than n items:
```

Comment:
If n is the number of items in a list, how could these list contains more than n items?

Reply:
Tried a better explaination here by adding `m` the number of list items and `n` the wanted number of displayed items.